static/app/components/avatarChooser.tsx

```diff
@@ -116,6 +116,11 @@ class AvatarChooser extends Component<Props, State> {
     const {model, dataUrl} = this.state;
 
     ev.preventDefault();
+    // before saving make sure user has selected an file
+    if (dataUrl === null) {
```

Comment:
Sorry for the late review, we don't need this anymore now that the button is disabled.

Reply:
Changes Done!